doc/contribute/index.rst

```diff
@@ -28,10 +28,18 @@ The CCF team welcome contributions to any part of the framework, including this
 
     Build CCF from source.
 
+    ---
+
+    :fa:`rocket` :doc:`release_policy`
```

Comment:
This seems like an odd page for this tile as release support is also shared across members, users and operators. Should this go on the main index, or under "overview"?

Reply:
I hesitated between here and overview, but overview seemed crowded, and if you squint a bit it's a project matter. I'm happy to switch it though, I don't feel strongly about it.

Reply:
Moved now